### DISCUSSION TITLE
A Solution to a problem I had: A JavaScript error occurred in the main process

### DISCUSSION THREAD

**Discussion by howebiz** [GetPublii/Publii#1643] [General]
I'm not sure if this is the right place to post this or if it's even worth sharing, but I'm leaving it here as a reference for the future.

I was having a hard time getting the app to work on my computer (Windows 10) due to an error during installation: "A JavaScript error occurred in the main process." The issue was that the default system folders, like 'C:\Users\Username\Documents,' are protected by default, which was preventing the installer from working properly in that location. I added an exception for 'Publii.exe' in the Windows settings, and the installation went smoothly afterward.

To do this, go to Ransomware Protection > Block history, identify the latest app that was blocked, click on Actions, and then select 'Allow on device.'

**Comment by paulkater** [comment]
![Selection_0005](https://github.com/user-attachments/assets/8ec267a3-934a-4fee-8be6-3e2337c9ffe5)

I have the same problem. Not always, but it's happened 3 or 4 times already.
Linux Ubuntu 22.04, AppImage Version: 0.46.3 (build 17008)
I saw a similar post which involved DropBox, I don't use it. The folders I'm using are not synced online.
Often, after restarting the program a few times, the error goes away.

**Reply by paulkater** [reply]
Intriguing. I KEPT having the problem today, until I messed with plugins a bit. For some reason the 'meta tab' plugin was on, switched that off. Page generation and sync started again without a problem. Could be a bad plugin, or a clash between plugins.
